source/documentation/how-we-work.html.md.erb

```diff
@@ -44,6 +44,36 @@ for how we prefer to work on code.
 ### Commit  Messages / Pull Requests
 We follow the [GDS Git Style Guide](https://github.com/alphagov/styleguides/blob/master/git.md).
 
+### Workflow
+
+Current (as of November 2021) workflow is:
+
+![workflow columns](/images/workflow-columns.png "diagram of the workflow columns")
+
+The purpose of each column are:
+
+**In Progress** - The ticket is currently being worked on.
+
+**Blocked**  - Unable to progress with the work due to some issues. Please add a comment explaining why the ticket is blocked and get help to unblock the ticket.
+
+**In Testable Editor** - This column is applicable only to the tickets related to the Editor app. The developer creates a Testable Editor based off of the code specific to their story. This generates a URL which can be visited for testing by any interested party. Developer should add the Testable Editor URL to the ticket.
```

Comment:
```suggestion
**In Testable Editor** - This column is applicable only to the tickets related to the Editor app. The developer creates a [Testable Editor](https://github.com/ministryofjustice/fb-editor/pull/476) based off of the code specific to their story. This generates a URL which can be visited for testing by any interested party. Developer should add the Testable Editor URL to the ticket.
```

Reply:
I've linked to the PR about how to set up a testable editor - might be good for future devs

Reply:
👍  thanks for that. have added a link to the testable editor description in the tech docs themselves